src/apps/mean-finance/mean-finance.definition.ts

```diff
@@ -0,0 +1,40 @@
+import { Register } from '~app-toolkit/decorators';
+import { appDefinition, AppDefinition } from '~app/app.definition';
+import { AppAction, AppTag, GroupType } from '~app/app.interface';
+import { Network } from '~types/network.interface';
+
+export const MEAN_FINANCE_DEFINITION = appDefinition({
+  id: 'mean-finance',
+  name: 'Mean Finance',
+  description:
+    'Our protocol enables users to Dollar Cost Average (DCA) any ERC20 into any ERC20 with their preferred period frequency.',
```

Comment:
Remove "our". The description should not be from the perspective of the protocol contributor, but rather as a third party. i.e.: "Mean Finance enables users to ..."